packages/button/src/mwc-button.ts

```diff
@@ -34,6 +34,9 @@ export class Button extends LitElement {
   @property({type: Boolean})
   dense = false;
 
+  @observer(function(this: LitElement, value: boolean) {
+    this.style.pointerEvents = value ? 'none' : '';
+  })
   @property({type: Boolean})
```

Comment:
Instead of adding an observer let's make the property `reflect: true` and in `mwc-button.scss` add:
```css
:host([disabled]) {
  pointer-events: none;
}
```

Reply:
Done